release-github.gradle

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+project.ext.githubUserName = project.hasProperty('githubUserName') ? githubUserName : ""
+project.ext.githubToken = project.hasProperty('githubToken') ? githubToken : ""
+project.ext.publishRepo = project.hasProperty('publishRepo') ? publishRepo : ""
+
+publishing {
+    repositories {
+        maven {
+            name = "GradleScriptsGitHubPackages"
+            url = uri(publishRepo)
+            credentials {
+                username = githubUserName
+                password = githubToken
+            }
+        }
+    }
+}
+
+tasks.create("releaseGithub") {
+    finalizedBy 'release'
+}
+
+releaseGithub {
+    project.ext.set("disableBintray", true)
+}
+
+if (project.hasProperty("disableBintray")) {
```

Comment:
There is no option to publish on bintray anymore. It's already turned off since 30th of March. I insist on removing it completely

Reply:
Fixed